content/md/products-super-power/enrich-your-reference-entity-records.md

```diff
@@ -0,0 +1,142 @@
+---
+id: enrich-your-reference-entity-records
+themes: products-super-power, manage-your-reference-entities
+title: "Enrich your reference entity records"
+popular: false
+ee-only: right
+related:
+---
+
+# Enrich your reference entity records
+
+## Add a record
+Once you’ve created a reference entity, you can add **records**.
+Let’s say that you have just created the reference entity `Brand`, you have defined its records attributes and now it is time to add new records. For example: Fermob, Kartell, Muuto, Fatboy…
+
+To add a new brand to the reference entity `Brand`:
+1. Go to your `Brand` entity (you are automatically on the records grid)
+1. Then click on the `Add new` button, choose a label for the record (the code is automatically generated but you can change it if needed).
+
+![Create a record](../img/Records_creation.png)
+
+::: tips
+If you have several records to add, you can stay in the creation pop-in to add quickly as many records as needed:
+1. Input the first record label
+1. Tick `Create another record`
+1. Press `Enter`
+1. Add the label for each record
+1. Once you've done, press `Esc` or `Echap`   
+:::
+
+::: info
+Note that we defined a limitation of 1 million records per reference entity to guarantee that the PIM is functional and runs smoothly.
+:::
+
+## Display records
+To see the records of the reference entity, click on the `Records` tab in the left menu. You will see the records grid, where all the records of the entity are listed.
+
+![Records grid](../img/Records_grid.png)
+
+### The records grid
+The records grid is quite different than the products' one. Thereafter are the novelties of this grid:
+- The line with the attributes names is frozen: it allows you to scroll down the page, while **continuing to see the name of the attributes**
+- The first 4 columns with image, label, code and completeness are also frozen. We added an **horizontal scroll** in order to see the content of all the columns
+- We also added a tooltip to **preview the content** of a text attribute directly in the grid
+- Finally, there is an **infinite scroll down** on this grid, which allows you to gather all the records in the same page.
+
+::: info
+In the records grid, you will see only certain types of attributes: code, text and image.
+:::
+
+::: info
+When you scroll down in the grid, only the first 500 results are displayed (in order to avoid performances issues). When you arrive at the end of the first 500 records, a message is displayed *"You just scrolled through the first 500 results out of 10005 records. To see more, try with more specific search criteria."*
```

Comment:
For me it's coming against what it is just written above: "- Finally, there is an **infinite scroll down** on this grid, which allows you to gather all the records in the same page."
it's not infinite if it's limited to 500 records :)

This limitation should have be written in the first sentence:
"- Finally, the grid can display up to 500 results using an infinite scroll. If you have more results, please filter your search. When reaching this limit, the grid displays the message "You just scrolled through the first 500 results out of 1450 records. To see more, try with more specific search criteria." "